src/inc/eventtracebase.h

```diff
@@ -374,7 +469,9 @@ namespace ETW
         VOID Append(SIZE_T currentFrame);
         EtwStackWalkStatus SaveCurrentStack(int skipTopNFrames=1);
     public:
+#if !defined(FEATURE_PAL)
         static ULONG SendStackTrace(MCGEN_TRACE_CONTEXT TraceContext, PCEVENT_DESCRIPTOR Descriptor, LPCGUID EventGuid);
+#endif //!defined(FEATURE_PAL)
         EtwStackWalkStatus GetCurrentThreadsCallStack(UINT32 *frameCount, PVOID **Stack);
```

Comment:
We should probably just ifdef out all of the call stack code since we're not bringing that over and much of it looks ETW specific.